code/game/objects/items/food/_food.dm

```diff
@@ -82,3 +85,9 @@
 	if(trash_type)
 		AddElement(/datum/element/food_trash, trash_type)
 	return
+
+///This proc makes things decompose. Set preserved_food to TRUE to make it never decompose.
+/obj/item/food/proc/MakeDecompose()
+	if(!preserved_food)
+		AddComponent(/datum/component/decomposition, decomp_flags = foodtypes)
+	return
```

Comment:
```suggestion
		AddComponent(/datum/component/decomposition, decomp_flags = foodtypes)
```

Reply:
Why remove the return? The other AddComponent() proc in _food.dm has it, so I thought it was needed.

```
/obj/item/food/proc/MakeGrillable()
	if(burns_on_grill)
		AddComponent(/datum/component/grillable, /obj/item/food/badrecipe, rand(20 SECONDS, 30 SECONDS), FALSE)
	return
```

Reply:
its a redundant return since its at the end of the proc and doesnt specify a return value. unless you specify a return value separate from . it will just return that. its not a big deal but i prefer not having redundant lines.

Reply:
alright, makes sense.